From b22fd73a59ca8d85e194ba49dcc233e53cb498cc Mon Sep 17 00:00:00 2001
From: Ievgen Mutavchi <Ievgen_Mutavchi@comcast.com>
Date: Fri, 25 Oct 2019 11:24:42 -0400
Subject: [PATCH] Fix build of WebCoreTestSupport

And add an option to inject internals object (disabled by default)
---
 Source/WebCore/platform/mock/PlatformSpeechSynthesizerMock.cpp   | 2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebCore/platform/mock/PlatformSpeechSynthesizerMock.cpp b/Source/WebCore/platform/mock/PlatformSpeechSynthesizerMock.cpp
index 67f97dd..5d3299b 100644
--- a/Source/WebCore/platform/mock/PlatformSpeechSynthesizerMock.cpp
+++ b/Source/WebCore/platform/mock/PlatformSpeechSynthesizerMock.cpp
@@ -77,7 +77,7 @@ void PlatformSpeechSynthesizerMock::cancel()
         return;
 
     m_speakingFinishedTimer.stop();
-    client()->speakingErrorOccurred(*m_utterance);
+    client()->speakingErrorOccurred(*m_utterance, SpeechErrorCanceled);
     m_utterance = nullptr;
 }
 
-- 
2.7.4

